src/Pmi/Service/SiteSyncService.php

```diff
@@ -147,14 +163,18 @@ public function syncAwardees()
             }
             $awardeesMap[$awardee->id] = $awardee->displayName;
         }
+        if (empty($awardeesMap)) {
+            throw new \Exception('No awardees found');
+        }
         $awardeesRepository = $this->em->getRepository('awardees');
         $awardeesRepository->wrapInTransaction(function() use ($awardeesRepository, $awardeesMap) {
             $awardeesRepository->truncate();
             foreach ($awardeesMap as $id => $name) {
-                $awardeesRepository->insert([
+                $awardeeId = $awardeesRepository->insert([
                     'id' => $id,
                     'name' => $name
                 ]);
+                $this->app->log(Log::AWARDEE_ADD, $awardeeId);
```

Comment:
Either because this is wrapped in a transaction or because there is no auto increment key on this table, `$awardeeId` is not set when I was testing. Either way, I think we should just log `$id` (the awardee ID) here.
```suggestion
                $this->app->log(Log::AWARDEE_ADD, $id);
```